Четность/Четность/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+namespace Parity
+{
+    class Program
+    {
+        static bool Parity(int m, int n)
+        {
+            return ((m % 2 == 0 && n % 2 != 0) || (n % 2 == 0 && m % 2 != 0));
```

Comment:
Модно было бы и проще
```suggestion
            return (m + n) % 2 != 0;
```